test/integration/auth.spec.ts

```diff
@@ -1912,6 +1932,7 @@ describe('admin.auth', () => {
         multiFactorConfig: deepCopy(expectedUpdatedTenant.multiFactorConfig),
         testPhoneNumbers: deepCopy(expectedUpdatedTenant.testPhoneNumbers),
         recaptchaConfig: deepCopy(expectedUpdatedTenant.recaptchaConfig),
+        emailPrivacyConfig: { enableImprovedEmailPrivacy: false },
```

Comment:
can you add this to expectedUpdatedTenant?

Reply:
`expectedUpdatedTenant` will yield `emailPrivacyConfig: {}` if the `enableImprovedPrivacy` variable is set to false. So we can't add this to that else the test will fail.